### DISCUSSION TITLE
[New example] Vite + Lit + DaisyUI

### DISCUSSION THREAD

**Discussion by txfs19260817** [saadeghi/daisyui#498] [Show and tell]
Hi, thank you for this awesome component library!

I wrote an [example](https://stackblitz.com/edit/vitejs-vite-x3rvrd?devtoolsheight=33&file=src/my-element.ts) which is a Vite + Lit + TypeScript application with DaisyUI. Since it is not easy to configure Tailwind CSS in Lit-Element, DaisyUI can be a great choice to design wonderful pages. 

I was wondering if there is any chance for me to contribute it to this repo's examples?

Cheers~!

**Comment by saadeghi** [comment]
Sure!  
Please add it to [/examples](https://github.com/saadeghi/daisyui/tree/master/examples) and submit a PR 🙌
